_episodes_rmd/07-plot-ggplot2.Rmd

```diff
@@ -123,7 +123,7 @@ With this many bars plotted, it's impossible to read all of the
 x-axis labels. A quick fix to this is the add the `coord_flip()` 
 function to the end of our plot code.
 
-```{r hist-subset-gapminder-flipped}
+```{r hist-subset-gapminder-flipped fig.cap = "Barplot showing GDP per capita. Country names on the y-axis are readable"}
```

Comment:
```suggestion
```{r hist-subset-gapminder-flipped, fig.cap = "Barplot showing GDP per capita. Country names on the y-axis are readable"}
```